saleor/graphql/order/types.py

```diff
@@ -715,6 +715,11 @@ class OrderLine(ModelObjectType[models.OrderLine]):
             "permissions to access." + ADDED_IN_39
         ),
     )
+    price_override = graphene.Field(
```

Comment:
Do you need this? I don't see that we're providing the `price_override` in `CheckoutLine` type

Reply:
Agree. It should be only on `CheckoutLine` model

Reply:
removed.